src/routes/safe/components/Apps/index.tsx

```diff
@@ -31,7 +31,7 @@ const StyledIframe = styled.iframe`
   box-sizing: border-box;
   width: 100%;
   height: 100%;
-  display: ${({ appIsLoading }) => (appIsLoading ? 'none' : 'block')};
+  display: ${({ hidden }: { hidden: boolean }) => (hidden ? 'none' : 'block')};
```

Comment:
why did you not use a generic? is there a specific reason?

Reply:
what do you mean by generic? to this `{hidden: boolean}` ? Because we are not using it in another place.

Reply:
styled.iframe<{ hidden?: boolean }>

Reply:
right, that's the correct way to do it, but why optional?

Reply:
I would say it should be false by default.

From my experience loading/hiding options are always optional

Reply:
yes, but not in this case, it's not a reusable component.